lib/v2/screens/onboarding/onboarding_screen.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:carousel_slider/carousel_controller.dart';
+import 'package:carousel_slider/carousel_options.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:dots_indicator/dots_indicator.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+import 'package:seeds/v2/screens/onboarding/components/pages/onboarding_page_1.dart';
+import 'package:seeds/v2/screens/onboarding/components/pages/onboarding_page_2.dart';
+import 'components/pages/onboarding_page_3.dart';
+import 'package:seeds/i18n/onboarding.i18n.dart';
+
+class OnboardingScreen extends StatefulWidget {
+  const OnboardingScreen({Key key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() {
+    return OnboardingState();
+  }
+}
+
+class OnboardingState extends State<OnboardingScreen> {
+  final CarouselController _controller = CarouselController();
+  int _selectedIndex = 0;
+
+  void onPageChange(int index, CarouselPageChangedReason changeReason) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
+  void onPageChangeForward(int index, CarouselPageChangedReason changeReason) {
+    if (index < 2) {
+      setState(() {
+        _controller.jumpToPage(index + 1);
+        _selectedIndex = index + 1;
+      });
+    }
+  }
+
+  void onPageChangeBackward(int index, CarouselPageChangedReason changeReason) {
+    if (index == 0) {
+    } else {
+      setState(() {
+        _controller.jumpToPage(index - 1);
+        _selectedIndex = index - 1;
+      });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
```

Comment:
`height` must be final, I think this build is execute more than once. is'n it?

Reply:
Also don't forgert make a pull to master before merge I removed a file :D

Reply:
Done both, thank you for the heads-up